app/javascript/react/components/FlavorTile.js

```diff
@@ -2,11 +2,9 @@ import React from "react";
 import { Link } from "react-router-dom";
 
 const FlavorTile = (props) => {
-  const { name } = props;
+  const { name, flavorId, regionId } = props;
   return (
-    <Link to={`/regions/${name}`}>
-      <p>{name}</p>
-    </Link>
+    <Link to={`/regions/${regionId}/flavors/${flavorId}`}>{name}</Link>
```

Comment:
unless you need a regionId for something, we can unnest this route

Reply:
Made suggested changes